src/fuse.js

```diff
@@ -84,6 +84,18 @@
     var key
     var keys
 
+    if (list.constructor === Array) this.typeof = 'array'
+    else this.typeof = typeof list;
+
+    if(this.typeof === 'object') {
+      list = Object.keys(list).map(function (i) {
```

Comment:
@boarnoah i think the mobile browsers do already have the `Object.keys` method.
However, I'm on a `for in` loop
```
    if(this.typeof === 'object') {
      var new_list = [];
      for (var key in list) {
        var modified = list[key];
        modified['__o_key'] = key;
        new_list.push(modified);
      }

      list = new_list;
    }
```